.env.test

```
@@ -0,0 +1,31 @@
+DOMAIN=localhost
+
+ENVIRONMENT=testing
+
+PROJECT_NAME="Kaapi-Guardrails"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)